lib/diagnostic.js

```diff
@@ -8,21 +8,21 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  return Math.min(1, 5);
 };
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum() {
-
+const sum = function sum(x) {
+  return x = 5 + 2;
 };
```

Comment:
Also, note that the function takes "zero or more arguments." How many does your take in?